hyperspy/drawing/utils.py

```diff
@@ -1697,3 +1699,106 @@ def picker_kwargs(value, kwargs=None):
         kwargs['picker'] = value
 
     return kwargs
+
+
+def plot_span_map(sig, spans=1):
+    """
+    Plot a span map.
+    
+    Plots a navigator consisting of the sum over all positions in sig, with `spans` spans. 
+
+    For each span a `BaseSignal` image is plotted consisting of integrated counts over the the range defined by the span.
+
+    The spans can be moved interactively and the corresponding images will update automatically.
+
+    Arguments
+    ---------
+    sig: hyperspy.signals.Signal1D
+        hyperspectra to inspect. Should have 2 spatial dimensions and 1 signal dimension.
+    spans: int, [hyperspy.roi.SpanROI]
+        spans that represent colour channels in map. Can either pass a list of `SpanROI` objects, or an `int`, `N`, in which case `N` `SpanROI`s will be created.
```

Comment:
```suggestion
        Spans that represent colour channels in map. Can either pass a list of `SpanROI` objects, or an `int`, `N`, in which case `N` `SpanROI`s will be created.
```

Could you format comments and docstrings with linebreaks after about 80 characters for readability of the file?

Reply:
Yes should be sorted now!